src/cli.ts

```diff
@@ -128,6 +128,7 @@ process.on('unhandledRejection', handler);
     useElectronClang: !!argv.useElectronClang,
     disablePreGypCopy: !!argv.disablePreGypCopy,
     projectRootPath,
+    skipPrebuilds: !!argv.skipPrebuilds,
```

Comment:
Does this new option also impact `prebuild` or just `node-pre-gyp`?

Reply:
It also appears to have a `buildFromSource` behavior rather than "skipping" the module entirely

Reply:
renamed `prebuild` -> `buildFromSource`